integration_test/ops_agent_test.go

```diff
@@ -4542,6 +4543,49 @@ func TestNoNvmlOtelReceiverWithoutGpu(t *testing.T) {
 	})
 }
 
+func TestPartialSuccess(t *testing.T) {
+	t.Parallel()
+	gce.RunForEachPlatform(t, func(t *testing.T, platform string) {
+		ctx, logger, vm := setupMainLogAndVM(t, platform)
+		logPath := logPathForPlatform(vm.Platform)
+
+		config := fmt.Sprintf(`logging:
+  receivers:
+    files_1:
+      type: files
+      include_paths: [%s]
+  service:
+    pipelines:
+      p1:
+        receivers: [files_1]`, logPath)
+
+		testFile, err := os.ReadFile(path.Join("partial_success", "test.log"))
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		if err = gce.UploadContent(ctx, logger, vm, bytes.NewReader(testFile), logPath); err != nil {
+			t.Fatal(err)
+		}
+
+		if err = agents.SetupOpsAgent(ctx, logger, vm, config); err != nil {
+			t.Fatal(err)
+		}
+
```

Comment:
Nit: It would be helpful to add a link to the public documentation describing the behaviour of which logs are going to be received or not by the API.